packages/@aws-cdk/aws-cognito/test/integ.user-pool-idp.google.ts

```diff
@@ -16,7 +16,7 @@ const userpool = new UserPool(stack, 'pool', {
 new UserPoolIdentityProviderGoogle(stack, 'google', {
   userPool: userpool,
   clientId: 'google-client-id',
-  clientSecret: 'google-client-secret',
+  clientSecretValue: SecretValue.unsafePlainText('google-client-secret-value'),
```

Comment:
@saleco can you add an integration test that tests using an actual secret stored in secrets manager? Some values in CloudFormation don't support resolving secret values so we want to make sure this one does.

Reply:
Thanks @corymhall 
Im working on it.

Reply:
@corymhall I have updated the integ test.
Unfortunately, I wasn't able to use my AWS account to execute it. 
Any help would be appreciated.
![image](https://user-images.githubusercontent.com/3430162/209234985-a56121aa-10c1-4ccd-ba9a-fee2a63a2435.png)